hack/extract-kubeconfig.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+namespace="${1:-}"
+test="${2:-}"
+if [[ -z "${namespace}" || -z "${test}" ]]; then
+	echo "USAGE: $0 <namespace> <test>"
+	exit 1
+fi
+
+function retrieve() {
+	pods="$( oc get pods --namespace ${namespace} --selector "ci.openshift.io/multi-stage-test=${test}" -o json )"
```

Comment:
Why pull this out of a pod, instead of grabbing it from the secret behind `$SHARED_DIR`?

Reply:
For example, even if we always expect a `test` pod to exist in each pipeline, pulling from the secret would allow us to get the kubeconfig during the install step(s) as well, for folks who wanted to monitor or jostle the cluster before the `test` pod exists.

Reply:
Well actually this would be the opposite -- grabbing it from the install pod would allow us to get it while installation was running, waiting for it to be in the secret would require install to already have finished.